### PR TITLE
feat: add example and test for bucket blueprint

### PR DESCRIPTION
**Pull request by bharathkkb**
Adds an example and test for bucket blueprint.
Keeping helpers in repo to iterate quickly, will be upstreamed to [build-tools](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/tree/master/infra/build/developer-tools/build/scripts) once stabilized.

### CODE REVIEW

**Review by morgante — APPROVED** [review]

File: .prow.yaml [hunk 1]
```diff
@@ -25,3 +25,25 @@ presubmits:
...
+    always_run: true
+    skip_report: true
+    decorate: true
+    cluster: build-blueprints
+    decorate_config:
+      timeout: 1h
+    spec:
+      serviceAccountName: int-test-sa
+      containers:
+        - image: gcr.io/cft-prow-img-test/cft/developer-tools-krm:1.3
```
Comment (bharathkkb) [inline .prow.yaml:38]: TODO @bharathkkb

Comment (bharathkkb) [reply .prow.yaml]: done

File: test/integration/simple-bucket/simple_bucket_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/krmt"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSimpleBucket(t *testing.T) {
+	bucketName := "test-bucket"
+	bucketTest := krmt.NewKRMBlueprintTest(t, krmt.WithSetters(map[string]string{"name": bucketName}))
```
[the remark this answers is not in the document]
Comment (bharathkkb) [reply test/integration/simple-bucket/simple_bucket_test.go]: done, forgot to remove after testing the upsert functionality

**Pull request merged**